packages/auth-construct/src/construct.ts

```diff
@@ -696,14 +697,29 @@ export class AmplifyAuth
       result.providersList.push('APPLE');
     }
     if (external.oidc) {
+      const oidc = external.oidc;
+      const requestMethod =
+        oidc.attributeRequestMethod === undefined
+          ? 'GET' // default if not defined
+          : oidc.attributeRequestMethod;
       result.oidc = new cognito.UserPoolIdentityProviderOidc(
         this,
         `${this.name}OidcIDP`,
         {
           userPool,
-          ...external.oidc,
+          attributeRequestMethod:
```

Comment:
if we do the `const oidc = {...external.oidc}`, we can do `...oidc` here to make it easier to read.

Reply:
we can't do ...oidc; we are remapping property types here